.github/workflows/cd_release.yml

```diff
@@ -0,0 +1,32 @@
+name: Release CD
```

Comment:
Do we really need a release CD workflow?

Reply:
This could probably be folded into the standard CD workflow, I just don't have enough experience and confidence so just copied things over from ES.